Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -1298,6 +1301,63 @@ bool qMRMLSubjectHierarchyTreeView::multiSelection()
 }
 
 //-----------------------------------------------------------------------------
+vtkIdType qMRMLSubjectHierarchyTreeView::firstSelectedInHierarchyFrom(vtkIdType itemID)
+{
+  // Check self
+  Q_D(qMRMLSubjectHierarchyTreeView);
+  if (d->SelectedItems.contains(itemID))
+    {
+    return itemID;
+    }
+
+  // Check children
```

Comment:
Let's make the code more readble. If you add a comment, make it actually explain something
"Look for item in children, recursively"